projects/elife.yaml

```diff
@@ -1693,6 +1693,32 @@ elife-libero-reviewer:
                 "{instance}-elife-libero-reviewer":
                     encryption: arn:aws:kms:us-east-1:512686554592:key/b6f44e77-46d8-41bb-a2e5-75c0588a2b20
 
+elife-libero-editor:
+    default-branch: null
+    domain: null
+    intdomain: null
+    aws:
+        ec2: false
+    aws-alt:
```

Comment:
@lsh-0 Wasn't sure what the difference between the `aws` and `aws-alt` keys are? At a guess wondered if the later is for Libero?

Reply:
`aws-alt` and `gcp-alt` are 'alternate configurations'.

What happens, roughly, is that the 'default' section at the top of the file has the top-level project section merged over it to give us our project configuration. We can create tweaked 'alt' versions of this project config under `aws-alt`. 

In your example, the S3 bucket could live in the regular project config like:

```yaml
elife-libero-reviewer:
    aws:
        ec2: false
        s3:
            "{instance}-elife-libero-reviewer":
                encryption: arn:aws:kms:us-east-1:512686554592:key/b6f44e77-46d8-41bb-a2e5-75c0588a2b20
    aws-alt:
        prod:
            s3:
                "{instance}-elife-libero-reviewer":
                    deletion-policy: retain
```

and then the `prod` environment would ensure the production version of that bucket isn't deleted when the stack is deleted. Or prod could have extra buckets or no buckets, etc.

You can see the final configuration using 

    ./bldr project.data

and picking a project, or in this case:

    ./bldr project.data:elife-libero-editor

default output format is Python, but you can also use JSON or YAML with:

    ./bldr project.data:elife-libero-editor,output_format=yaml

etc.

From what I can see in your proposal the whole thing could live in the top-level with no alt configs:

```
elife-libero-editor:
    aws:
        ec2: false
        s3:
            "{instance}-elife-libero-editor": {}
        sqs:
            # Monitor the production process bucket for new deposits from Exeter.
            elife-libero-editor-import--{instance}:
                subscriptions:
                    - "elife-production-processes"
        docdb: {}
```

I would recommend production buckets have an explicit deletion policy though. If it's ok to delete production data, have `deletion-policy: delete` set.